src/app/wallet/views/transactions.nim

```diff
@@ -132,6 +133,41 @@ QtObject:
   proc sendTransaction*(self: TransactionsView, from_addr: string, to: string, assetAddress: string, value: string, gas: string, gasPrice: string, password: string, uuid: string) {.slot.} =
     self.sendTransaction("transactionSent", from_addr, to, assetAddress, value, gas, gasPrice, password, uuid)
 
+  proc transferEth*(self: TransactionsView, from_addr: string, to_addr: string, value: string, gas: string, gasPrice: string, password: string, uuid: string): bool {.slot.} =
+    try: 
+      if not validate(HexDataStr(from_addr)): return false
+      if from_addr.len != 42: return false
```

Comment:
Could we do a `validateAddress` funciton that does both `validate` and the length check?